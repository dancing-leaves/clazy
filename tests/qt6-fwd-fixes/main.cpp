#include <QtCore/qglobal.h>

class myClass
{
  class QStringList; // this is ignored
};

template <typename Key, typename T> class QCache  ;     class SomeClass;
template <typename Key, typename T> class QHash  ;
template <typename Key, typename T> class QMap /*blabla*/ ;
template <typename Key, typename T> class QMultiHash;
template <typename Key, typename T> class QMultiMap;
class QPair;
template <typename T> class QQueue;
template <typename T> class QSet;
template <typename T> class QStack;
template <typename T, qsizetype Prealloc = 256> class QVarLengthArray;
template <typename T> class QList;
template<typename T> class QVector;
class QStringList;
class QByteArrayList;
class QMetaType;
class QVariant;
class QVariantList;
class QVariantMap;
class QVariantHash;
class QVariantPair;
