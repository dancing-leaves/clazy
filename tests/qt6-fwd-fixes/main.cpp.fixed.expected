#include <QtCore/qglobal.h>

class myClass
{
  class QStringList; // this is ignored
};

#include <QtCore/qcontainerfwd.h>
class SomeClass;
